docs/articles_en/about-openvino/contributing.rst

```diff
@@ -0,0 +1,169 @@
+the `docs/articles_en <https://github.com/openvinotoolkit/openvino/tree/master/docs/articles_en>`__
+folder, using `Sphinx <https://www.sphinx-doc.org/>`__ and the
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`__
+markup language.
+
+OpenVINO `developer documentation <https://github.com/openvinotoolkit/openvino/tree/master/docs/dev>`__
+is available only in markdown in the `docs/dev <https://github.com/openvinotoolkit/openvino/tree/master/docs/dev>`__
+folder.
+
+To edit docs, consider using the Editor’s
+`guide <https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING_DOCS.md>`__
+and contacting `documentation maintainers <https://github.com/orgs/openvinotoolkit/teams/openvino-docs-maintainers>`__,
+who will help you with information architecture and formatting, as well as
+review, adjust, and merge the PR.
+
+.. rubric:: Review user documentation
+   :name: review-user-documentation
+
+In most cases, creating a PR is enough to correct a documentation mistake, improve
+the language, and update or extend the information. For your convenience, the
+top-right panel of most pages includes the “Edit on GitHub” button that will
+take you to the source file of the given article.
+
+.. rubric:: Write new content
+   :name: write-new-content
+
+For more extensive changes in docs, reach out to any of the
+`documentation maintainers <https://github.com/orgs/openvinotoolkit/teams/openvino-docs-maintainers>`__
+to discuss the new content.
+
+
+:fas:`bullhorn` Promote and support OpenVINO
+############################################
+
+.. rubric:: Popularize OpenVINO
+   :name: popularize-openvino
+
+Articles, tutorials, blog posts, demos, videos, and any other involvement in the
+OpenVINO community is more than welcome. If you discuss or present OpenVINO on
+various social platforms, you are raising awareness of the product among A.I.
+enthusiasts and enabling other people to discover the toolkit.
+
+Feel free to reach out to OpenVINO developers if you need help with making a
+contribution. You can also contact
+`documentation maintainers <https://github.com/orgs/openvinotoolkit/teams/openvino-docs-maintainers>`__
+, if you need help with visuals, brand materials, or content creation in general.
+
+.. rubric:: Help other community members
+   :name: help-community
+
+If you are an experienced OpenVINO user and want to help, you can share your
+expertise with the community at any time. Check GitHub
+`Discussions <https://github.com/openvinotoolkit/openvino/discussions>`__
+and `Issues <https://github.com/openvinotoolkit/openvino/issues>`__ to see if
+you can help someone.
+
+.. note::
+
+   By contributing to the OpenVINO project, you agree that your contributions
+   will be licensed under `the terms of the OpenVINO repository <https://github.com/openvinotoolkit/openvino/blob/master/LICENSE>`__.
+
+
+Additional Resources
+#####################
+
```

Comment:
https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md#Technical-guide could also be added.

Reply:
> https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md#Technical-guide could also be added.

The article below, Code Contribution Guide is based on the technical guide you mention :)